lib/mayaUsd/ufe/UsdTransform3dMatrixOp.cpp

```diff
@@ -355,6 +355,18 @@ Ufe::ScaleUndoableCommand::Ptr UsdTransform3dMatrixOp::scaleCmd(double x, double
         UsdTimeCode::Default());
 }
 
+Ufe::SetMatrix4dUndoableCommand::Ptr UsdTransform3dMatrixOp::setMatrixCmd(const Ufe::Matrix4d& m)
+{
+    return nullptr;
+}
+
+void UsdTransform3dMatrixOp::setMatrix(const Ufe::Matrix4d& m) { _op.Set(toUsd(m)); }
```

Comment:
This is how little work parent -absolute has to do for matrix op prims: just set their (local) matrix.